src/AppBundle/Entity/User.php

```diff
@@ -406,7 +406,7 @@ public function getUsername()
      */
     public function getEmail()
     {
-        return $this->email;
+        return strtolower($this->email);
```

Comment:
If we're _always_ treating the email as lowercase in the frontend, does it make sense to do the same in API? Or even to always store it in lowercase when we populate the data?

Reply:
I agree - I think we should lowercase all emails at the point they are persisted, and that way they will always travel around the application as lowercase. We then only need to convert any user input used for lookup (password reset, admin search).

Reply:
Yes Ive done this, needs further approval. The setEmail method was always using strtolower() which is why there are no instances of capitals in the database. but for consistency Ive changed the DTOs too